### PR TITLE
:scroll: Fix ChangeLog: document all the changes. [skip ci]

### PR DESCRIPTION
**Pull request by kaikai1024**
Document all the changes to this project of all versions from v0.21.0

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -42,8 +42,106 @@ All notable changes to this project will be documented in this file. And this pr
...
+- [Doc] Build a new [documentation website](https://docs.citahub.com/en-US/cita/cita-intro). [@zhouyun-zoe]
+
+### Tool
+
+- [Optimization] Split util module into standalone crates. [@yangby]
+- [Refactor] Combing the snapshot logic and rewrite snapshot_tools. [@keroro520]
+
+[v0.21.0]: https://github.com/cryptape/cita/compare/v0.21...HEAD
+
+[@KaoImin]: https://github.com/KaoImin
```
Comment (yangby-cryptape) [inline ChangeLog.md]: We should try to keep the list of links alphabetized (`[A-Za-z]`).

Comment (kaikai1024) [reply ChangeLog.md]: That's better.

Comment (yangby-cryptape) [reply ChangeLog.md]: - For [Vim]ers: `:133,$ sort u`.
- For [Emacs]ers: Select the region, then `M-x sort-lines`.

[Vim]: https://www.vim.org/
[Emacs]: https://www.gnu.org/software/emacs/

Comment (kaikai1024) [reply ChangeLog.md]: > * For [Vim](https://www.vim.org/)ers: `:133,$ sort u`
> * For [Emacs](https://www.gnu.org/software/emacs/)ers:
>   
>   * Select the region:
>     
>     * `M-x goto-line`
>     * `133`
>     * `C-@`
>     * `M->`
>   * Sort: `M-x sort-lines`

so sweet. :heart:

**Pull request merged**